### ISSUE TITLE
Max2Babylon roughness map ignored?

### ISSUE THREAD

**Issue by echadwick-wayfair** [BabylonJS/Exporters#775]
Exporter seems to be ignoring Roughness texture. Has something changed in the last month or so?

Max2Babylon 202020228.3
3ds Max 20202.3

![image](https://user-images.githubusercontent.com/40177036/75591357-8810b780-5a4d-11ea-8dca-912a739a679b.png)

![image](https://user-images.githubusercontent.com/40177036/75591320-75967e00-5a4d-11ea-9871-56839ec63d2f.png)

[torusknot_rough.zip](https://github.com/BabylonJS/Exporters/files/4269320/torusknot_rough.zip)

**Comment by echadwick-wayfair** [comment]
Also the AO doesn't seem to export unless Merge AO Map is disabled. Probably because the Roughness map isn't getting exported, so the ORM can't be packed.

These are 24bit PNGs, no alpha.

![2020-02-28 17_07_11-Babylon js - Export scene to babylon or glTF format v20200228 3](https://user-images.githubusercontent.com/40177036/75591453-c6a67200-5a4d-11ea-9e5e-87a15b46f943.jpg)

**Comment by Drigax** [comment]
Exported as glTF, the ORM map looks properly created and merged:
![image](https://user-images.githubusercontent.com/3952729/75737461-9e6c7c80-5cb4-11ea-9a6c-5952a34a2171.png)

**Comment by Drigax** [comment]
Looks like my fix for #781, #785 fixed this:

![image](https://user-images.githubusercontent.com/3952729/76486300-31f02c80-63dc-11ea-99a6-a04c410621ca.png)

Part of the fix addressed an issue where our packed textures are now given a unique name based on their components, and I fixed an issue where we were improperly excluding some textures from export. Looks like everythings working now, but feel free to reopen if this is not the case :)

**Comment by Drigax** [comment]
I did discover that part of the reason why the texture path was missing was due to a change in our parameter intialization caused by #760, I don't think we should be ANDing the checkbox enable state with its value, as we auto-check the disabled checkboxes for a reason. This logic caused `ExportParameters.WriteTextures` and `ExportParameters.OverwriteTextures` to be initialized to `false` when exporting glb, which caused our textures to be excluded.

I still see some issues with the output .glb scene, however:

When loaded into the babylon sandbox we can see that the torus has a proper material assigned:
![image](https://user-images.githubusercontent.com/3952729/76495319-d5007080-63f3-11ea-80ba-b9bef337da4f.png)

And the material is setup to reference the packed and included ORM texture:
![image](https://user-images.githubusercontent.com/3952729/76495421-11cc6780-63f4-11ea-89de-f6e9e6c67014.png)

But the texture does not appear to be properly rendered...
![image](https://user-images.githubusercontent.com/3952729/76495692-9cad6200-63f4-11ea-9d80-9b48199ca50a.png)

I'm not sure how to proceed here. I'm inclined to believe that the glTF is formatted properly, but I'm not sure what may be going wrong here.

@bghgary, does this seem like a loader issue? I was able to generate this attached glb, but the AO still does not properly render. If one manually replaces the texture in-engine with the included texture, it renders as expected.

[Export.zip](https://github.com/BabylonJS/Exporters/files/4322734/Export.zip)
